configs/settings-example.ini

```diff
@@ -5,7 +5,7 @@
 [general]
 sitename = "Hitchwiki"
 metanamespace = "Hitchwiki"
-debug = false
+debug = true
```

Comment:
The change is ok by me. Let's keep it in one settings file to keep things simple.